tests/unit/fs_events_test.ts

```diff
@@ -49,7 +49,7 @@ Deno.test(
   { permissions: { read: true, write: true } },
   async function watchFsBasic() {
     const testDir = await makeTempDir();
-    const iter = Deno.watchFs(testDir);
+    using iter = Deno.watchFs(testDir);
```

Comment:
Why was this needed?

Reply:
So that the watcher gets closed at the end of the test.

Reply:
To clarify, why does removing `Deno.FsWatcher.prototype.return()` cause the `FsWatcher` to not be automatically closed. I guess the method is automatically closed once the iterator is done...